templates/telemetry/modules/modules.4.parquetmr.txt

```diff
@@ -0,0 +1,32 @@
+message modules {
+  optional binary clientId (UTF8);
+  required binary creationDate (UTF8);
+  required binary id (UTF8);
+  required binary type (UTF8);
+  required double version;
+  required group payload {
+    required int64 version;
+    optional group modules (LIST) {
+      repeated group list {
+        optional group element (TUPLE) {
```

Comment:
Should this be a `TUPLE`? See here for an explanation of lists: https://github.com/apache/parquet-format/blob/master/LogicalTypes.md

It seems to me this should just be a normal group with 4 fields.

Reply:
Done; fixed!

Reply:
fyi: TUPLE is our extension and is not documented at the link above see: https://bugzilla.mozilla.org/show_bug.cgi?id=1355553